dist/parcoords.standalone.js

```diff
@@ -8766,7 +8770,7 @@
     var axisDots = function axisDots(config, pc, position) {
       return function (_r) {
         var r = _r || 0.1;
-        var ctx = pc.ctx.marks;
+        var ctx = pc.ctx.dots;
```

Comment:
corresponding line of source code:
https://github.com/BigFatDog/parcoords-es/blob/5959a719ec0447ce6864712ebfabe881ac65b85c/src/api/axisDots.js#L7